tests/functionality_tests/test_activation_shapes.py

```diff
@@ -25,15 +52,15 @@ def main():
         rec_col_limit  = 2 * cols // 3## Good value 22
         grid.activate_rectangle(rec_row_start, rec_col_start, rec_row_limit, rec_col_limit)
         grid.find_border_nodes()
-        way_points = grid.get_all_lawnmower_waypoints_adjustable()
+        way_points = grid.get_all_guided_lawnmower_waypoints_adjustable(is_vertical)
 
     if activation_type== "circle":
         circle_center_row = rows // 2 ## Good value 19
         circle_center_col = cols // 2 ## Good value 17
         circle_radius = min(rows, cols) // 2 ## Good value 17
         grid.activate_circle(circle_center_row, circle_center_col, circle_radius)  
```

Comment:
Sammie did leave a comment earlier saying that we should no longer ask users for row/col input and instead fix it to 34 rows, 38 cols. Right now, the users can set a row/col input (ex. 60,60)  so that the center becomes out-of-bounds. 

<img width="663" alt="Screen Shot 2023-03-03 at 4 50 23 PM" src="https://user-images.githubusercontent.com/40185030/222836291-f9714f55-0e90-43e2-8024-62a495999262.png">

Although cases like this will be resolved once the row/cols are fixed, there's an edge case where the center could be 30,30 (in-bounds) and the radius is 20. Should this case be allowed? If not, then we need to account for it here.

Reply:
I think it should be allowed but I am just wondering what the red nodes are. If they are border nodes shouldn't that be incorrect? Maybe when Andrew fixes the border function it will work.

Reply:
Thank you Shreya, this is very interesting. We should definitely discuss this during the meeting. If we do want to consider cases this this, we would need to fix the initialization.

Reply:
The circle case is bc we are allowing user inputs. Should be resolved when we remove input space since we are fixing the matplotlib to the eng quad size
Explained here:
https://github.com/cornellnexus/src/pull/108#discussion_r1134750910